### DISCUSSION TITLE
Coalesce usage for edge aggregation clarification

### DISCUSSION THREAD

**Discussion by cameronosmith** [pyg-team/pytorch_geometric#2387] [Q&A] [answered]
Hi, I'm writing a custom clustering module, and want to aggregate the edge index as performed in EdgePool, such that for every node `v` in the formed cluster `c`, `c` is adjacent to all the nodes `v` was adjacent to (in their respective clusters). 
Am I correct in that aggregating the edge index is as easy as you wrote in the EdgePool module: `new_edge_index, _ = coalesce(cluster[edge_index], None, N, N)` ?

**Comment by rusty1s** [comment, marked as answer]
That is correct :)

**Reply by cameronosmith** [reply]
Thank you
